spring-cloud-gcp-data-datastore/src/test/java/org/springframework/cloud/gcp/data/datastore/it/DatastoreIntegrationTests.java

```diff
@@ -845,15 +850,34 @@ public void testPageableGqlEntityProjectionsSlice() {
 		assertThat(testEntityProjections.get(0).getColor()).isEqualTo("blue");
 	}
 
-	//An index containing both "color" and "size" is required
-	@Ignore
-	@Test
-	public void testSliceString() {
-		Slice<String> slice =
-				this.testEntityRepository.getSliceStringBySize(2L, PageRequest.of(0, 3));
+	@Test(timeout = 10000L)
+	public void testSliceString() throws IOException, URISyntaxException, InterruptedException {
+		URL resource = this.getClass().getResource("/index.yaml");
+		File file = Paths.get(resource.toURI()).toFile();
+
+		Process process = new ProcessBuilder("gcloud", "datastore", "indexes", "create", file.getAbsolutePath(), "-q")
+				.start();
+
+		if (process.waitFor() != 0 ){
+			throw new RuntimeException("Error while creating index.");
+		}
+
+		Slice<String> slice;
+		try {
+			slice = this.testEntityRepository.getSliceStringBySize(2L, PageRequest.of(0, 3));
+		}
+		catch (DatastoreException e) {
+			if (e.getMessage().contains("no matching index found")) {
+				throw new RuntimeException(
+						"The required index is not found. "
```

Comment:
I don't think you need the try/catch to print an extra message for the user; it would be just as clear to add a 1-liner comment that the test may fail if running for the first time since it takes time to create the index.